rust-executor/src/entanglement_service/mod.rs

```diff
@@ -0,0 +1,93 @@
+use std::{collections::HashSet, env, fs::File, io::{BufReader, Write}, path::{self, Path}, sync::Mutex};
+
+use crate::{agent::{sign, sign_string_hex, AgentService}, graphql::graphql_types::EntanglementProof};
+
+pub(crate) mod entanglement_service_extension;
+
+lazy_static! {
+  static ref ENTANGLEMENT_PROOF_FILE: Mutex<String> = Mutex::new(format!("{}/entanglement_proof.json", env::var("APPS_DATA_PATH").unwrap_or_else(|_| ".".to_string())));
```

Comment:
<insert comment on env var from above>

Reply:
my bad I think we change this to use the SQL database